xarray/core/common.py

```diff
@@ -925,6 +934,7 @@ def _resample(
         offset: pd.Timedelta | datetime.timedelta | str | None,
         origin: str | DatetimeLike,
         restore_coord_dims: bool | None,
+        boundaries: Literal["exact", "trim"] | None = None,
```

Comment:
shall we use `drop_incomplete: bool` like in SeasonResampler? Note that you could be missing data in the middle too...